backend/src/index.js

```diff
@@ -40,114 +41,123 @@ BODY
 
 */
 app.post("/translate", async (req, res) => {
-    const { words } = req.body;
-
-    if(words && words.length) 
-    {
-        const wordList = words.split(" ");
-        const urls = wordList.map(l => [l, urlBuilder(l)]);
+  const { words } = req.body;
 
-        const paths = (await Promise.all(urls.map(([word, url]) => downloadVideo(url, word)))).filter(e => !!e);
-        const parsedPaths = paths.map(path => `\"${path}\"`);
-        const thingToSendToPython = `[${parsedPaths.join(",")}]`;
+  if (words && words.length) {
+    const wordList = words.split(" ");
+    const urls = wordList.map((l) => [l, urlBuilder(l)]);
 
-        const pyRes = await callPythonScript(thingToSendToPython);
-        const linkToMergedVid = `http://localhost:5000/${pyRes}`;
+    const paths = (
+      await Promise.all(urls.map(([word, url]) => downloadVideo(url, word)))
+    ).filter((e) => !!e);
 
-        res.status(200).send({
-            link: linkToMergedVid
-        });
+    if (paths.length == 0) {
+      return res.status(400).json({ message: "Bad request" });
     }
+
+    const parsedPaths = paths.map((path) => `\"${path}\"`);
+    const thingToSendToPython = `[${parsedPaths.join(",")}]`;
+    const pyRes = await callPythonScript(thingToSendToPython);
+    const linkToMergedVid = `http://localhost:5000/${pyRes}`;
```

Comment:
This will break shit if the internal port is changed. Might as well throw it in a .env or construct it somehow via ENV vars